src/pudl/transform/ferc1.py

```diff
@@ -3942,34 +3860,3 @@ def plants_steam_ferc1(
         transformed_fuel=fuel_ferc1,
     )
     return convert_cols_dtypes(df, data_source="ferc1")
-
-
-if __name__ == "__main__":
```

Comment:
The issue mentioned converting this to use dagster, but I propose we just move this and encourage using the dagster ui to iterate on transformers

Reply:
Or the `materialize_asset` script we made!